tests/compile-fail/weak_memory/racing_mixed_size_read.rs

```diff
@@ -0,0 +1,37 @@
+// ignore-windows: Concurrency on Windows is not supported yet.
+// compile-flags: -Zmiri-ignore-leaks
```

Comment:
We don't even complete program execution here, so this flag should be unnecessary.